store/db_test.go

```diff
@@ -78,6 +78,9 @@ func createTable(db *DB, t *testing.T) {
 }
 
 func TestDBRace(t *testing.T) {
+	// TODO(sgotti) this will not find concurrenct accesses to ql db from
+	// multiple processes using multiple goroutines. A test that spawns at
```

Comment:
@jonboulle I changed to commit comment removing the very confusing description and adding an example.